src/server.c

```diff
@@ -1871,17 +1871,17 @@ void createSharedObjects(void) {
     shared.noscripterr = createObject(OBJ_STRING,sdsnew(
         "-NOSCRIPT No matching script. Please use EVAL.\r\n"));
     shared.loadingerr = createObject(OBJ_STRING,sdsnew(
-        "-LOADING Redis is loading the dataset in memory\r\n"));
+        "-LOADING Server is loading the dataset in memory\r\n"));
     shared.slowevalerr = createObject(OBJ_STRING,sdsnew(
-        "-BUSY Redis is busy running a script. You can only call SCRIPT KILL or SHUTDOWN NOSAVE.\r\n"));
+        "-BUSY Server is busy running a script. You can only call SCRIPT KILL or SHUTDOWN NOSAVE.\r\n"));
```

Comment:
At AWS we have this hardcoded. there are other places this stuff seems hardcoded: https://github.com/netty/netty/blob/fee3c39d0835ec18d7aace8776db5c5952ce8272/codec-redis/src/main/java/io/netty/handler/codec/redis/FixedRedisMessagePool.java#L44

I think this need to be more thoughtfully considered for compatibility. I'm leaning towards we just accept this as a compatibility problem forever.

Reply:
Agree, don't change error replies. We'll not do that in the compat release.

Besides, I have a PR for those specifically: #206